angularjs-portal-home/src/main/webapp/my-app/layout/controllers.js

```diff
@@ -105,6 +107,17 @@ define(['angular', 'jquery'], function(angular, $) {
                 }
             };
 
+            //Check guestMode
+            var that = this;
+            that.user = [];
+            $scope.notGuestMode = true;
```

Comment:
I would prefer to have `$scope.guestMode` to remove the possibility of a double negative of not not being a guest ;)